resources/puppetlabs/lein-ezbake/template/global/install.sh.erb

```diff
@@ -70,9 +71,15 @@ function task_service {
         else
             task install_source_rpm_systemd
         fi
-    elif [ "$OSFAMILY" = "Debian" ]; then
+      elif [ "$OSFAMILY" = "Debian" ]; then
+        sysv_codenames=("squeeze" "wheezy" "lucid" "precise" "trusty")
+        codename=`facter os.distro.codename`
```

Comment:
It's only used that way for source installs though. That code path is not invoked for the native package builds afaik.